mrun

```diff
@@ -1,3 +1,2 @@
 #! /bin/sh
-
-exec python "mesos_scheduler.py" --mpi "$@"
+exec python "$PARACEL_INSTALL_PREFIX/mesos_scheduler.py" --mpi "$@"
```

Comment:
`PARACEL_INSTALL_PREFIX` is set to its right value inside `prun.py`. `mrun` can be only called inside `prun.py`, I mean they are always in the same process with same env. So don't worry about the logic. Anyway, I will try to use `dirname`.